tests/algorithms/column_pooler_test.py

```diff
@@ -101,7 +101,7 @@ def testLearnSinglePattern(self):
     self.init()
 
     object = self.generateObject(1)
-    self.learn(object, numRepetitions=1, newObject=True)
+    self.learn(object, numRepetitions=2, newObject=True)
```

Comment:
Thanks. I don't want to gratuitously override parameters in tests just to make tests pass. The issue you pointed out could affect a number of simulations and experiments. I'm ok with making numRepetitions=2 or (ideally) improving the parameters so all simulations are more robust. For example, perhaps we should make initialProximalPermanence=0.61 in the column pooler file itself or some other change that could make everything better.